ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -134,6 +135,15 @@ vsphere)
     sed -i "/secret-name \=/c user = \"${GOVC_USERNAME}\"" "$VSPHERE_CONF_FILE"
     sed -i "/secret-namespace \=/c password = \"${GOVC_PASSWORD}\"" "$VSPHERE_CONF_FILE"
     export TEST_PROVIDER=vsphere;;
+alibabacloud)
+    mkdir -p ~/.ssh
+    cp "${CLUSTER_PROFILE_DIR}/ssh-privatekey" ~/.ssh/kube_alibaba_rsa || true
+    export PROVIDER_ARGS="-provider=alibabacloud -gce-zone=us-east-1"
+    # TODO: make openshift-tests auto-discover this from cluster config
+    REGION="$(oc get -o jsonpath='{.status.platformStatus.alibabacloud.region}' infrastructure cluster)"
+    export TEST_PROVIDER="{\"type\":\"alibabacloud\",\"region\":\"${REGION}\",\"multizone\":true,\"multimaster\":true}"
```

Comment:
Dropping `export` for now might [avoid][1].

```
E0124 22:54:06.535535     281 test_context.go:503] Unknown provider "alibabacloud". The following providers are known: aws azure baremetal gce gke ibmcloud kubemark kubevirt local openstack ovirt skeleton vsphere
```

until origin can be taught about the new provider type.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/20841/rehearse-20841-pull-ci-openshift-installer-master-e2e-alibaba/1485730605567053824

Reply:
https://github.com/openshift/origin/pull/26782